OpenSim/Common/Component.h

```diff
@@ -1554,6 +1554,7 @@ template <class T> friend class ComponentMeasure;
             _connectorsTable[connector.getName()] = ix;
         }
     }
+protected:
```

Comment:
As a note, the current comment would not appear in doxygen. As this method will now be part of the developer interface, I agree it should appear in doxygen.